modules/internal/DefaultRectangular.chpl

```diff
@@ -1285,29 +1285,63 @@ module DefaultRectangular {
       }
     }
 
-    override proc dsiReallocate(bounds:rank*range(idxType,BoundedRangeType.bounded,stridable)) {
-      dsiReallocate(bounds, bounds);
+    override proc dsiReallocate(allocBound: range(idxType,
```

Comment:
Why not use the same naming scheme across overloads, e.g. `allocBound` -> `bounds` or vice-versa?

Reply:
I wanted to differentiate the two arguments here - one is about the bound of the allocation, the other is about the bound of the logical array.  I could change `bounds` to `allocBound` if you think it would be worthwhile.  That would require updates in ChapelDistribution.chpl, DefaultRectangular.chpl, BlockDist.chpl, CyclicDist.chpl, StencilDist.chpl, and possibly some tests since they all need to use the same names to be valid overloads.  Think it's worth it?